### ISSUE TITLE
easy handle already used in multi handle

### ISSUE THREAD

**Issue by benmarwick** [Ironholds/WikipediR#7]
I often get this error:

```
Error in function (type, msg, asError = TRUE)  : 
  easy handle already used in multi handle
```

For example, after a bunch of these: `page_info("en","wikipedia", page = "Aaron Halfaker")`

I'm guessing this is some kind of rate-limiting from the API. What's best way to get `page_info` for, say 10,000 pages without running into this error? Thanks!

**Comment by Ironholds** [comment]
Actually it's not! It's one of the weirdnesses in RCurl, exposed through httr - handle overflows. Hopefully httr 1.0.0 (which depends instead on Jeroen's 'curl') should fix this. If you want to grab the development branch from Hadley's repo that'd be a fix until 1.0.0 goes on CRAN (to be fair, that happens on the 24th)

**Comment by benmarwick** [comment]
Ok, thanks, yes I've updated httr and that error has gone. Thanks again!
